presage/src/manager/registered.rs

```diff
@@ -271,18 +271,19 @@ impl<S: Store> Manager<S, Registered> {
         let (pre_keys_offset_id, next_signed_pre_key_id, next_pq_pre_key_id) = account_manager
             .update_pre_key_bundle(
                 &mut self.store.clone(),
+                ServiceIdType::AccountIdentity, // TODO: What about PNI?
```

Comment:
This one is tricky. Can't find anything equivalent in Signal Desktop.

Reply:
I just compared to the previous impl of  `update_pre_key_bundle`. It was doing it only for `AccountIdentity` keys. So, at least the behavior is the same.

Reply:
From my look at the Signal Desktop code (function `maybeUpdateKeys` of `AccountManager`, both seem to be updated sometimes (in `UpdateKeysListener` and `AccountManager.setPni`). Not sure though if any of those apply to presage though, which only seems to update those after setup, while SD does it every two days as far as I can read.

Reply:
This is correct. When PNI is fully implemented, pre key generation should also get triggered for the PNI account, but this retains the previous behaviour.

Reply:
I have updated the TODO to change that once it is fully implemented.